src/AcceptanceTests/When_using_databus_with_client.cs

```diff
@@ -41,7 +41,8 @@ public EndpointWithCustomClient()
                 EndpointSetup<DefaultServer>(config =>
                 {
                     config.UseDataBus<AzureDataBus>()
-                        .UseBlobContainerClient(SetupFixture.BlobContainerClient);
+                        .Container(SetupFixture.ContainerName)
```

Comment:
In theory we could introduce a ConfigureDatabus class that sets the default container for all tests so that no test would ever forget to set it